runtimes/protocol/getAvailableServerConfigurations.ts

```diff
@@ -0,0 +1,25 @@
+import { ProtocolRequestType } from './lsp'
+
+export const getAvailableServerConfigurationsRequestType = new ProtocolRequestType<
+    GetAvailableServerConfigurationsParams,
+    any[],
+    never,
+    void,
+    void
+>('aws/getAvailableServerConfigurations')
+
+export interface GetAvailableServerConfigurationsParams {
```

Comment:
Config params already exists https://microsoft.github.io/language-server-protocol/specifications/lsp/3.17/specification/#workspace_configuration. Can we use/extend those instead of creating a custom one ourselves?

Reply:
I think we should not mix arguments with existing LSP config params, as intent is slightly different. We probably won't need `scopeUri`, and `section` values might not overlap. Also we'll have more flexibility evolving the payload separately from standard LSP, if needed